.github/workflows/trigger-caraml-doc-sync.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow triggers sync of docs to caraml-dev/docs
+
+name: Trigger CaraML Docs Sync
+
+on:
+  push:
+    branches: 
+    - main 
+    tags:
+    - v*
+  pull_request:
+
+jobs:
+  trigger-sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger Remote Doc Sync Workflow
+        uses: caraml-dev/docs/.github/actions/trigger-remote-docs-sync@main
+        with:
+          module: 'model'
+          git_https: 'https://github.com/gojek/merlin.git'
+          doc_folder: 'docs'
+          branch: 'main'
```

Comment:
May I know why you only use `main` branch here instead of the job can be triggered by `on.push.branches` and `on.push.tags`?

I think we should only trigger the docs sync when a new tag/release is released because development can still happen in the main branch which means the number of changes introduced will be more compared to using only the tag release. If you are agree, we can rename `ref` as the field key and use `github.ref` as the value as suggested in the documentation: https://docs.github.com/en/actions/learn-github-actions/contexts#github-context. Then you might need to update the actions in caraml-dev/docs as well.

Reply:
I have set it to tag only.